### ISSUE TITLE
Unable to drag RAIL application window to second monitor connected during app runtime

### ISSUE THREAD

**Issue by Voxnod** [FreeRDP/FreeRDP#10699]
When running a RAIL application via an X11 client with the `/multimon` argument and initially having only one monitor connected, the app window cannot be moved to a second monitor if it is plugged in after the application has already started. You can visually drag the window to the newly connected monitor, but once you release the left mouse button, the window jumps back to the first monitor. It behaves as though the `/multimon` argument was never passed.

Video Demonstration: 
https://github.com/user-attachments/assets/1325df67-88c1-4ab2-bd1a-fd628dc900a3

For comparison, I tested this with the Windows RDP client, and it allows dragging a remote app window to a newly connected monitor without issue.

Video Demonstration:

https://github.com/user-attachments/assets/0c3aca32-ba59-4ef5-b0b5-db546d147059

I am using FreeBSD 14.1 with Plasma 5.

**Comment by akallabeth** [comment]
yes, we did not implement the display channel updates for dynamic monitor resizing.
pull requests welcome :)
